djangocms_icon/static/djangocms_icon/js/libs/bootstrap-iconpicker.js

```diff
@@ -397,6 +405,11 @@
         this.select(op.icon);
     };
 
+    Iconpicker.prototype.setVersion = function (value) {
```

Comment:
can you also add here a comment that this is custom code added to the library and required (as we could upgrade the library in the future)